build.sbt

```diff
@@ -123,8 +124,8 @@ lazy val scala = Project(id = "content-atom-model", base = file("scala"))
       (scroogeUnpackDeps in Compile).value.flatMap { dir => (dir ** "*.thrift").get }
     },
     libraryDependencies ++= Seq(
-      "org.apache.thrift" % "libthrift" % "0.9.2",
-      "com.twitter" %% "scrooge-core" % "4.5.0"
+      "org.apache.thrift" % "libthrift" % "0.9.3",
+      "com.twitter" %% "scrooge-core" % "4.12.0"
```

Comment:
Although it should not be a blocker, as if there is still an incompatibility, we will continue to downgrade in content-api models to `libthrift 0.9.1`